### ISSUE TITLE
Contrast is too low on orange text

### ISSUE THREAD

**Issue by sarah37** [ieee-vgtc/ieeevis.org#2369]
The orange color (used for headers and links) doesn't have sufficient contrast with the white background of the website. For the bolded headers it passes WCAG contrast checks but not for the regular body text. When hovering over the links the color turns even more pale. I would suggest updating this with a darker color. `#b25f2f` would be a darker version of the shade of orange that just about passes contrast checks. 

Screenshot from [https://colourcontrastchecker.com/](https://colourcontrastchecker.com/): 
<img width="1008" height="765" alt="Image" src="https://github.com/user-attachments/assets/bba137e8-562e-4e64-a66b-865d16ce923c" />

**Comment by mcnuttandrew** [comment]
This is a good point and, despite the slow response, is definitely on my mind. I feel like the most accessible approach would be to not just make the orange be darker, but try to prefer to use black text as much as possible? 

Using highlight colors puts some firm requirements on what colors can be used in each year's palette, so such a design change might also make palette picking a little simpler.
